src/components/Contentful/CardRow.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<section-with-background-classic
+		:background-content="background"
+		:vertical-padding="verticalPadding"
+	>
+		<template
+			#content class="tw-bg-primary"
+		>
+			<kv-page-container>
+				<kv-grid
+					class="tw-mx-auto tw-grid-cols-12"
+				>
+					<div
+						v-for="(storyCard, index) in storyCards"
+						:key="index"
+						:class="`tw-col-span-12 md:tw-col-span-${12/storyCards.length}`"
```

Comment:
I tried to do something l similar in a recent PR. Once the code got up to development the tailwind class didn't work due to the tw-class name being split. We went with this method instead: 
:class="{
							'tw-col-span-12': true,
							'md:tw-col-span-6': contentfulMediaItems.length >= 2,
							'lg:tw-col-span-6': contentfulMediaItems.length === 2,
							'lg:tw-col-span-4': contentfulMediaItems.length === 3,
							'lg:tw-col-span-3': contentfulMediaItems.length === 4,
						}"
That way complete tailwind classes are pass through and respected on render.

Reply:
That's a good call. It's surprisingly working now... Probably b/c of your code lolz. I'll update with the more specific classes.